### ISSUE TITLE
Inconsistent dropdown content

### ISSUE THREAD

**Issue by outdoor2kode** [WordPress/wordcamp.org#1292]
### Describe the bug

Based on other dropdowns and other latest themes, the title shouldn't be there inside the dropdown.

### Expected behavior

Remove the text 'Format'.

### Screenshots / Screencasts

![Screenshot 2024-04-23 at 13 09 53](https://github.com/WordPress/wordcamp.org/assets/18050944/5cba62bb-9c7e-4558-a40a-735b02f05428)

![Screenshot 2024-04-23 at 13 10 01](https://github.com/WordPress/wordcamp.org/assets/18050944/e8c96c20-e6b6-4e4e-8e2b-b62fb28d36ce)

![Screenshot 2024-04-23 at 13 13 09](https://github.com/WordPress/wordcamp.org/assets/18050944/03364275-0f10-4136-a5fa-942a38c32c16)

**Comment by ryelle** [comment]
It looks like "Format" is a single-select, which intentionally has a different look/behavior to it — the visible title and lack of clear button (#1293) are intentional here. See https://github.com/WordPress/pattern-directory/issues/648#issuecomment-2031738949. You can see the same behavior on Showcase & Patterns:

<img width="479" alt="" src="https://github.com/WordPress/wordcamp.org/assets/541093/69093287-d5c3-48f7-9173-6f5822b71234">

<img width="534" alt="" src="https://github.com/WordPress/wordcamp.org/assets/541093/ceb64ef2-b865-428b-a08a-7e77563cae3e">

**Comment by outdoor2kode** [comment]
Thank you for sharing the rationale behind this! Feel that a single-select isn't suitable here because sorting doesn't filter out content, yet the Format reduces the visible content. Users would likely want to clear the filter. Or alternatively, like on Patterns, we could also add an "All" option and set it as the default.

**Comment by pkevan** [comment]
> Thank you for sharing the rationale behind this! Feel that a single-select isn't suitable here because sorting doesn't filter out content, yet the Format reduces the visible content. Users would likely want to clear the filter. Or alternatively, like on Patterns, we could also add an "All" option and set it as the default.

I was just about to comment to say the same thing 😄

**Comment by pkevan** [comment]
I think we can get the same `All` option like patterns by using the filter: https://github.com/WordPress/pattern-directory/blob/trunk/public_html/wp-content/themes/wporg-pattern-directory-2024/inc/block-config.php#L13.

**Comment by ryelle** [comment]
Yep, these options are set up here: https://github.com/WordPress/wordcamp.org/blob/3d425fd9cac3f52b843125f4f3fe1decff709a3a/public_html/wp-content/themes/wporg-events-2023/inc/events-query.php#L445-L470

So you can add an "All" option there.

**Comment by dd32** [comment]
I don't think adding 'All' here is ideal, even if consistent with other uses of it.

We should extend the single-dropdown to allow having a clear option, as this is just weird:

https://github.com/user-attachments/assets/957fdfc2-9dbc-4117-b5bf-ce9943159459
